shell/platform/tizen/channels/text_input_channel.cc

```diff
@@ -111,8 +133,20 @@ static void InputPanelStatChangedCallback(void* data,
   }
 }
 
-static Eina_Bool RetrieveSurroundingCallback(void* data, Ecore_IMF_Context* ctx,
-                                             char** text, int* cursor_pos) {
+void TextInputChannel::InputPanelGeometryChangedCallback(
+    void* data, Ecore_IMF_Context* context, int value) {
+  if (!data) {
+    LoggerD("[No Data]\n");
+    return;
+  }
+  TextInputChannel* self = (TextInputChannel*)data;
+  self->UpdateCurrentKeyboardGeometry();
```

Comment:
I think you can remove the `UpdateCurrentKeyboardGeometry` function as it's only used here.

Reply:
I fixed it